src/Core/Framework/DataAbstractionLayer/Event/PartialEntityLoadedEvent.php

```diff
@@ -7,9 +7,6 @@
 use Shopware\Core\Framework\DataAbstractionLayer\PartialEntity;
 use Shopware\Core\Framework\Log\Package;
 
-/**
- * @internal
```

Comment:
should be final or?

So I wouldn't want to assume that it should be expandable or extendable

Reply:
I will change this ;)

Reply:
This is for you to decide, what is final :)